core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/workflow/WorkflowRewriter.scala

```diff
@@ -0,0 +1,424 @@
+package edu.uci.ics.texera.workflow.common.workflow
```

Comment:
Throughout this file, the method names are not very well defined. Also, the arguments are ambigious as well: `id`, `opID`, `operatorID`, etc. It's better we can unify method and argument names to make the file more readable.